src/platform/android/mod.rs

```diff
@@ -261,7 +266,12 @@ impl Window2 {
     }
 
     #[inline]
-    pub fn set_fullscreen(&self, _state: FullScreenState) {
+    pub fn set_fullscreen(&self, _monitor: Option<RootMonitorId>) {
+    }
+
+    #[inline]
+    pub fn get_current_monitor(&self) -> RootMonitorId {
+        unimplemented!()
```

Comment:
Android provides only one dummy monitor. This function should return `RootMonitorId(MonitorId)`.